gui/colors/adjbases.py

```diff
@@ -876,10 +876,10 @@ def render_background_cb(self, cr, wd, ht):
         cr.set_line_width(self.OUTLINE_WIDTH)
         cr.stroke()
 
-        cr.rectangle(b, b, int(eff_wd / 2), eff_ht)
+        cr.rectangle(b, b, int(eff_wd // 2), eff_ht)
         cr.set_source_rgb(*curr.get_rgb())
         cr.fill()
-        cr.rectangle(wd / 2, b, eff_wd - int(eff_wd / 2), eff_ht)
+        cr.rectangle(wd // 2, b, eff_wd - int(eff_wd // 2), eff_ht)
```

Comment:
Our Cairo drawing often uses 0.5px offsets to make 1px lines align to the pixel grid. The worst that can result is a fuzzy line though.